dataset/imagenet.py

```diff
@@ -0,0 +1,148 @@
+import tensorflow as tf
+import os
+
+
+class ImageNet:
+
+  TFRECS_FORMAT = {
+    'image' : tf.io.FixedLenFeature([],tf.string),
+    'height': tf.io.FixedLenFeature([], tf.int64),
+    'width': tf.io.FixedLenFeature([], tf.int64),
+    'filename' : tf.io.FixedLenFeature([],tf.string),
+    'label': tf.io.FixedLenFeature([], tf.int64),
+    'synset': tf.io.FixedLenFeature([], tf.string)
+  }
+
+  _MEAN = tf.constant([0.485, 0.456, 0.406])
+  _STD = tf.constant([0.229, 0.224, 0.225])
+
+  _EIG_VALS = tf.constant([[0.2175, 0.0188, 0.0045]])
+  _EIG_VECS = tf.constant([
+      [-0.5675, 0.7192, 0.4009],
+      [-0.5808, -0.0045, -0.8140],
+      [-0.5836, -0.6948, 0.4203],
+      ])
+
+
+  def __init__(self,tfrecs_filepath = None,
+               batch_size = 128,
+               image_size = 224,
+               augment_fn = 'default'):
+
+    if tfrecs_filepath is None:
+      raise ValueError('List of TFrecords paths cannot be None')
+    print('hi')
+    self.tfrecs_filepath = tfrecs_filepath
+    self.batch_size = batch_size
+    self.image_size = image_size
+    self.augment_fn = augment_fn
+
+  
+  def _read_tfrecs(self):
```

Comment:
Would suggest adding docstrings for all cases.

Reply:
Is it okay if I add them a bit later? I am maintaining a doc with all the documentation to add later, but for now I think it would be better to keep the code clean, at least until we know that all of this works well on GCP.

Reply:
It makes the review a bit harder. So, I would suggest adding the docstrings at this phase.

Reply:
Okay